src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -0,0 +1,192 @@
+import {promptsUpdate} from 'app/actionCreators/prompts';
+import robotLogo from 'app/../images/robot-logo.png';
+import SentryTypes from 'app/sentryTypes';
+import {snoozedDays} from 'app/utils/promptsActivity';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import withApi from 'app/utils/withApi';
+
+const DUMMY_COMMIT = {
+  id: 'ff721e0',
+  author: {name: 'Uh/Oh'},
+  dateCreated: moment()
+    .subtract(3, 'day')
+    .format(),
+  repository: {
+    provider: {id: 'integrations:github', name: 'GitHub', status: 'active'},
+  },
+  message: t('This commit accidentally broke something'),
+};
+
+class EventCauseEmpty extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    organization: SentryTypes.Organization.isRequired,
+    project: SentryTypes.Project.isRequired,
+  };
+
+  state = {
+    shouldShow: undefined,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  fetchData() {
+    const {api, project, organization} = this.props;
+
+    api.request('/promptsactivity/', {
+      method: 'GET',
+      query: {
+        project_id: project.id,
+        organization_id: organization.id,
+        feature: 'suspect_commits',
+      },
+      success: data => {
+        this.setState({
+          shouldShow: this.shouldShow(data),
+        });
+      },
+      error: err => {
+        Sentry.captureException(err);
+      },
+    });
+  }
+
+  shouldShow({data} = {}) {
+    if (data && data.dismissed_ts) {
+      return false;
+    }
+    if (data && data.snoozed_ts) {
+      return snoozedDays(data.snoozed_ts) > 7;
+    }
+    return true;
+  }
+
+  handleClick(action) {
+    const {api, project, organization} = this.props;
+
+    const data = {
+      projectId: project.id,
+      organizationId: organization.id,
+      feature: 'suspect_commits',
+      status: action,
+    };
+    promptsUpdate(api, data).then(this.setState({shouldShow: false}));
+  }
+
+  render() {
+    const {shouldShow} = this.state;
+    if (!shouldShow) {
+      return null;
+    }
+
+    return (
+      <div className="box">
+        <StyledPanel dashedBorder>
+          <BoxHeader>
+            <Description>
+              <h3>{t('Suspect Commits')}</h3>
+              <p>{t('Identify which commit caused this issue')}</p>
+            </Description>
+            <ButtonList>
+              <Button
+                size="small"
+                priority="primary"
+                href="https://docs.sentry.io/workflow/releases/#create-release"
+              >
+                {t('Read the docs')}
+              </Button>
+
+              <div>
+                <Tooltip title={t('Remind me in a week')}>
+                  <SnoozeButton
+                    size="small"
+                    onClick={() => this.handleClick('snoozed')}
+                    data-test-id="snoozed"
+                  >
+                    {t('Later')}
```

Comment:
i'm not sure users will understand the difference between "later" and "dismiss". "remind me later" is more explicit -- did we run out of space for it? if so, how about keeping it remind me later, but replacing dismiss with an X?

Reply:
ya, it was a space issue. we tried using an X but it didn't look as good, and landed on adding the tooltip. i can see about getting "remind me later" back

<img width="308" alt="Screen Shot 2019-07-11 at 4 19 35 PM" src="https://user-images.githubusercontent.com/16394317/61091629-bdea2880-a3f7-11e9-9b08-cc0239ab1f2c.png">

Reply:
I'm personally okay with how that reads ^